### PR TITLE
add dashboard layout

### PR DESCRIPTION
**Pull request by nhathoang241199**
**Dashboard layout**
image

Dashboard layout include:

1. Sidebar
image

2. Panel
image

3. AccountInfo
image

And banner component: (isn't belong to layout) 
image

### CODE REVIEW

**Comment by satoshiman** [comment]
2 icon day/night mode & send (I don't know the purpose) do not need white background 
image

It's better if we use `monospace font` for wallet address format
image

Add `GAKI` unit after the balance (or Your balance => Your GAKI balance)

Change `Faucet` CTA => `Request testnet GAKI` 

It seems responsive wasn't applied to this layout?

File: src/routes.tsx [hunk 1]
```diff
@@ -25,86 +25,60 @@ import SponsoredPool from './pages/SponsoredPool';
...
+import featureFlags from 'components/FeatureFlags';
+
+const { checkFeature, EFeatureFlag } = featureFlags;
 
 const isDisplayGameCreatorFeature = checkFeature(EFeatureFlag.GameCreator);
 
 const dashRoutes = [
   {
     path: '/dashboard',
     name: 'Dashboard',
```
Comment (lhtrung307) [inline src/routes.tsx:37]: Should translate these too.

**Review by lhtrung307 — CHANGES_REQUESTED** [review]

File: src/layouts/DashboardLayout/DashboardLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const DashboardLayout: React.FC = ({ children }) => (
+  <Box sx={dashBoardStyled}>
+    <SideBar />
+    <VStack
+      flex={10}
+      sx={{
+        background: 'greyBg',
+        display: 'flex',
+        flexDirection: 'column',
```
Comment (lhtrung307) [inline src/layouts/DashboardLayout/DashboardLayout.tsx:18]: Redundant style.

File: src/layouts/DashboardLayout/components/AccountInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    if (currentAccount) {
+      try {
+        unsubscribe = await api?.query.system.account(
+          acctAddr(currentAccount),
+          (balance: any) => {
+            setAccountBalance(balance.data.free);
+          }
+        );
+      } catch (error) {
+        console.log('error: ', error);
```
Comment (lhtrung307) [inline src/layouts/DashboardLayout/components/AccountInfo.tsx:57]: Show a toast for this.

**Review by lhtrung307 — APPROVED** [review]

**Pull request merged**